rosidl_generator_rs/resource/Cargo.toml.em

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 rosidl_runtime_rs = "0.3"
 serde = { version = "1", optional = true, features = ["derive"] }
+serde-big-array = "0.5.1"
```

Comment:
@fawdlstty this should be marked as optional (like `serde`) as well

Additionally, so that `serde-big-array` gets installed when `serde` is, you can add the following:

```[features]
serde = ["dep:serde", "dep:serde-big-array"]
```

Reply:
fixed